src/IO/ReadHelpers.h

```diff
@@ -1159,8 +1159,11 @@ inline bool tryReadText(IPv6 & x, ReadBuffer & buf) { return tryReadIPv6Text(x,
 inline void readText(is_floating_point auto & x, ReadBuffer & buf) { readFloatText(x, buf); }
 
 inline void readText(String & x, ReadBuffer & buf) { readEscapedString(x, buf); }
-inline void readText(LocalDate & x, ReadBuffer & buf) { readDateText(x, buf); }
+
 inline void readText(DayNum & x, ReadBuffer & buf) { readDateText(x, buf); }
+inline void readText(DayNum & x, ReadBuffer & buf, const DateLUTImpl & time_zone) { readDateText(x, buf, time_zone); }
```

Comment:
Could we have just one function (maybe with default argument) instead of overloads l. 1163 and l. 1164.